### PR TITLE
Add dynamic mobile menu using JavaScript pair programming project

### PR DESCRIPTION
**Pull request by juaryimami**
In this project:
- Dynamic menu added using JavaScript.
- `Readme` updated based on the project requirement.
-  we practiced well about pair programming.
   Thank you for reviewing our project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,226 +1,375 @@
...
+			<ul class="navbar__menu__ul ul flex">
+			  <li class="navbar__menu__li">
+				<a href="#works" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">Portfolio</a>
+			  </li>
+			  <li class="navbar__menu__li">
+				<a href="#about" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">About</a>
+			  </li>
+			  <li class="navbar__menu__li">
+				<a href="#contact" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">Contact</a>
+			  </li>
```
Comment (thecodechaser) [inline index.html:40]: - [x] You did a great job on the styles but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design as much as possible. Please fix the below issue and make sure your design matches the Figma design.
  - Please make sure the menu options are starting on the same line vertically.
  - Please apply the blur effect in the background. `hint: backdrop-filter: blur(Npx);` N is referred to as a number for pixels.

### Your design
image

### Figma design
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @juaryimami👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (juaryimami) [reply index.html:40]: ok I will fix it soon. thank you.

Comment (TedLivist) [inline index.html:40]: - [ ] While you have implemented the blur effect you are yet to align your menu options vertically as requested.

  port

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @juaryimami,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (juaryimami) [reply index.html:40]: ok thank you.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @juaryimami,

✔️ Implementation of blur effect

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**